aspnet/DropDownList/Customization.md

```diff
@@ -29,9 +29,9 @@ It provides the short description of the expected value in dropdown and will dis
         
 {% endhighlight %}
 
-![ ](Customization_images/Customization_img1.jpeg)
+![Customization in ASPNET DropDownList](customization_images/aspnet-dropdownlist-customization.jpeg)
```

Comment:
ASP.NET [dot is missing] Check Platform name in all places.

Reply:
done